sdk/src/main/java/io/radar/sdk/RadarSettings.kt

```diff
@@ -213,6 +215,19 @@ internal object RadarSettings {
         getSharedPreferences(context).edit { putString(KEY_FOREGROUND_SERVICE, foregroundJson) }
     }
 
+    internal fun getVerificationSettings(context: Context): RadarVerificationSettings? {
+        val optionsJson = getSharedPreferences(context).getString(KEY_VERIFICATION_SETTINGS, null) ?: return null
+        val optionsObj = JSONObject(optionsJson)
+        return RadarVerificationSettings.fromJson(optionsObj)
+    }
+
+    internal fun setVerificationSettings(context: Context, options: RadarVerificationSettings?) {
+        val optionsObj = options?.toJson()
+        val optionsJson = optionsObj?.toString()
+        getSharedPreferences(context).edit { putString(KEY_VERIFICATION_SETTINGS, optionsJson) }
```

Comment:
we probably don't want to set this for 99% of our customers (right now) who don't use location verification, so let's add an `if` check to guard against this